modules/sbo-setting-annotations-mapping-optional.adoc

```diff
@@ -0,0 +1,30 @@
+:_content-type: PROCEDURE
+[id="sbo-setting-annotations-mapping-optional_{context}"]
+= Setting annotations mapping to be optional
+
+You can have optional fields in the annotations. For example, a link to the credentials might not be there if the service endpoint does not require authentication. In such cases, a field might not exist in the target path of the annotations. As a result, {servicebinding-title} generates an error, by default. 
+
+As a service provider, to indicate whether you require annotations mapping, you can set a value to the `optional` flag in your annotations while enabling services. {servicebinding-title} provides annotations mapping only if the target path is available. When the target path is not available, the {servicebinding-title} skips providing the annotations mapping. 
+
+.Procedure
+
+* To enable {servicebinding-title} consider a new field in the annotations, provide the mapping by setting the `optional` flag value to `true`:
```

Comment:
```suggestion
* To make a new field in the annotations optional set the `optional` flag value to `true`:
```

Reply:
@pmacik Thanks! I like this. Just a small change from my side:
"To make a field in the annotations optional, set the `optional` flag value to `true`:"

Reason: It is for all fields and not just new fields.

Reply:
Done